prExplorer/Application/Products/Delete.cs

```diff
@@ -30,7 +30,7 @@ public async Task<Result<Unit>> Handle(Command request, CancellationToken cancel
 
                 if (user == null)
                 {
-                    // Eğer kullanıcı bulunamazsa hata işleme.
+                    //Error handling if the user is not found.
                     return Result<Unit>.Failure("Kullanıcı bulunamadı.");
```

Comment:
Lets make this english. We should also return failure msg based on restaurant language. So maybe backend json file for errors.

Reply:
Checked and edited.

Reply:
https://matteocontrini.medium.com/consistent-error-responses-in-asp-net-core-web-apis-bb70b435d1f8
lets create error controller. 
We can get user from IUserAccessor from main controller. 
Carry user to new controller where we can detect its language.
Since our new controller will detect and provide custom message for it. We carry ObjectResult to main controller and pass it to user. 
We need to add user language code to AppUser entity.

*-Easier way would be get language code from the token itself.

Reply:
Take into consideration that you may still need to translate some error messages on the server-side. Imagine you will have some server-side process and you need to log an error to a text log file on the server. In that case, you won't send it to the client. The language of the user should be available in both Web API (web part) and server

Reply:
Maybe its good to add userLanguage on AppUser entity. And when using Serilog we will find a way to use userLanguage on there as well.